structuredefinitions/Extension-UKCore-ConditionBodyStructure.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-ConditionBodyStructure" />
+  <url value="http://hl7.org/fhir/6.0/StructureDefinition/Extension-UKCore-ConditionBodyStructure" />
+  <version value="1.0.0" />
```

Comment:
being in draft and not through a C&TA sprint I'd like this, and the others, to be 0.0.1 as per https://semver.org/spec/v0.1.0.html#spec-item-5